draft-ietf-acme-acme.md

```diff
@@ -994,7 +995,7 @@ Link: <https://example.com/acme/some-directory>;rel="index"
 
 If the client wishes to update this information in the future, it sends a POST
 request with updated information to the account URI.  The server MUST ignore any
-updates to the "key", or "order" fields or any other fields it does not
+updates to the "key" and "order" fields. The server MUST ignore, and any other fields it does not
```

Comment:
My concern w/ the original wording is that it I parse it as or(key, order, other fields)-[it does not know] -- but it knows key and order.